### ISSUE TITLE
What is this operation after complete maxtrials?

### ISSUE THREAD

**Issue by KevinRyu** [keras-team/autokeras#1348]
Hello.

I run the following python script in jupyter notebook.
There was no error up to final trials. And I was able to see "Trial 10 Complete".
By the way, progress is started after completing max trials.
What is this operation? (I attached text capturing from jupyter cell. Please see the "2. Messages from AutoKeras")
It takes so long time. And I think that this progress proceed up to epoch number...
Is this operation normal?

**1. My Autokeras codes for regression:**
early_stopping = EarlyStopping(monitor='val_loss', patience=10, mode='min')

input_node1 = ak.Input()
input_node2 = ak.Input()
input_node3 = ak.Input()
input_nodes = ak.Merge(merge_type='concatenate')([input_node1, input_node2, input_node3])

dense_node = ak.DenseBlock()(input_nodes)

output_node = ak.RegressionHead(output_dim=1, loss=tf.keras.losses.MSE, metrics=['mse', 'accuracy'])(dense_node)

auto_model = ak.AutoModel(inputs=[input_node1, input_node2, input_node3], outputs=output_node, overwrite=True, max_trials=10, tuner="greedy", seed = 1)

auto_model.fit([train_x, train_a1, train_a2], train_y, batch_size=32, epochs = 1000, callbacks=[early_stopping])

===============================================================================
**2. Messages from AutoKeras**

Trial 10 Complete [00h 02m 04s]
val_loss: 2.0365827083587646

Best val_loss So Far: 0.00017729321552906185
Total elapsed time: 00h 54m 31s
INFO:tensorflow:Oracle triggered exit
Epoch 1/1000
8980/8980 [==============================] - 17s 2ms/step - loss: 0.0087 - mse: 0.0087 - accuracy: 0.0062
Epoch 2/1000
8980/8980 [==============================] - 17s 2ms/step - loss: 0.0031 - mse: 0.0031 - accuracy: 0.0063
Epoch 3/1000
8980/8980 [==============================] - 17s 2ms/step - loss: 0.0022 - mse: 0.0022 - accuracy: 0.0063
Epoch 4/1000
8980/8980 [==============================] - 17s 2ms/step - loss: 0.0020 - mse: 0.0020 - accuracy: 0.0063
Epoch 5/1000
8980/8980 [==============================] - 17s 2ms/step - loss: 0.0018 - mse: 0.0018 - accuracy: 0.0063

**Comment by haifeng-jin** [comment]
Please refer to this: https://github.com/keras-team/autokeras/discussions/1344
